bolt-service/Makefile

```diff
@@ -0,0 +1,25 @@
+all: image
```

Comment:
@davejohnston  You probably know better than I - can this be a docker-compose file? If not why not?